.changeset/tidy-impalas-refuse.md

```diff
@@ -0,0 +1,7 @@
+---
+"@comet/admin-date-time": major
+---
+
+Remove `clearable` prop from `FinalFormDatePicker`
```

Comment:
Technically it was removed from the `DatePicker`, which happens to affect the usage of `FinalFormDatePicker` and `DateField`. 

```suggestion
Remove `clearable` prop from `DatePicker`
```

or

```suggestion
Remove `clearable` prop from `DatePicker`, `DateField` and `FinalFormDatePicker`
```